v2/sourcedb-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/failureinjectiontesting/SrcDbToSpannerMySQLSevereErrorNonShardedFT.java

```diff
@@ -119,14 +118,16 @@ public void dataflowWorkerFailureTest()
         ChainedConditionCheck.builder(
                 List.of(
                     SpannerRowsCheck.builder(spannerResourceManager, AUTHORS_TABLE)
-                        .setMinRows(900)
-                        .setMaxRows(900)
+                        .setMinRows(90)
+                        .setMaxRows(90)
                         .build(),
                     SpannerRowsCheck.builder(spannerResourceManager, BOOKS_TABLE)
-                        .setMinRows(900)
-                        .setMaxRows(900)
+                        .setMinRows(90)
+                        .setMaxRows(90)
                         .build(),
-                    new SevereErrorsCheck(pipelineLauncher, jobInfo, 100)))
+                    DlqEventsCountCheck.builder(gcsResourceManager, "output")
```

Comment:
Would it be possible to  change this to a path to the DLQ folder within the `output` directory?
There could be more folders added in future and referring to `dlq` or `dlq/severe` folder might future proof the test case.

Reply:
Done.